javascript/javascript3/week1/WeatherApp/index.js

```diff
@@ -0,0 +1,118 @@
+const input = document.getElementById('city_name');
+const button = document.getElementById('btn');
+const msg = document.getElementById('errorMsg')
+
+button.addEventListener('click', () => {
+    if (input.value) {
+        const cityName = input.value.toLowerCase();
+        url = 'https://api.openweathermap.org/data/2.5/weather?q=' + cityName + '&appid=19a22c19916855a97938c4084c52be58';
+        fetch(url)
+            .then(response => response.json())
+            //.then(data=>console.log(data))
+            .then(data => renderCityWeatherInfo(data));
+    } else {
+        msg.innerHTML = 'Please enter the city';
+        return false;
```

Comment:
I don't think you need to return anything here. 
```suggestion
```

Reply:
okay